go/engine/identify2_test.go

```diff
@@ -665,6 +665,36 @@ func TestIdentify2NoSigchain(t *testing.T) {
 	}
 }
 
+// TestIdentifyDeleted tests that identifying a deleted user gives the correct
+// error
+func TestIdentify2Deleted(t *testing.T) {
+	tc := SetupEngineTest(t, "Identify2Deleted")
+
+	u := CreateAndSignupFakeUserWithUser(tc, "login")
+	userId := u.User.GetUID()
+	DeleteAccount(tc, u)
+
+	tc.Cleanup()
+	tc = SetupEngineTest(t, "Identify2Deleted2")
+	defer tc.Cleanup()
+
+	i := newIdentify2WithUIDTester(tc.G)
+	tc.G.Services = i
+	arg := &keybase1.Identify2Arg{
+		Uid: userId,
+	}
+	eng := NewIdentify2WithUID(tc.G, arg)
+	ctx := Context{IdentifyUI: i}
+
+	err := eng.Run(&ctx)
```

Comment:
I'm migrating this test to my branch and I think this is the problem here...you need to call engine.RunEngine(eng, ctx) to run an engine, not the Run method.